training/ItemTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ItemTableViewController.swift
+//  training
+//
+//  Created by Benedykt Ziobro on 07/11/2017.
+//  Copyright © 2017 Benedykt Ziobro. All rights reserved.
+//
+
+import UIKit
+import HandySwift
+
+class ItemTableViewController: UITableViewController {
+    
+    var items: [String] = Array(repeating:"", count:10)
+    var isTrigeringTask = false
+    
+    let celIdentifier = "ItemTableViewCell"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.estimatedRowHeight = 44.0 
+        isTrigeringTask = true
+        trigereTask()
+        generateData()
+       
+    }
+    
+    override func didReceiveMemoryWarning() {
+        isTrigeringTask=false
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: celIdentifier, for: indexPath) as? ItemTableViewCell else {
+            fatalError("Cell is wrong type. Should be the " + celIdentifier)
```

Comment:
fatalError isn't gentle way - crash

Reply:
Ok I fix it, now I only log error using NSLog